File: Common/InMotionGIT.Common/MetadataCatalogAttributes/EntitySpanishDesigner.vb
Namespace Attributes

    ''' <summary>
    ''' Define el titulo o nombre completo en español  asociado a la clase.
    ''' </summary>
    ''' <remarks></remarks>
    Public NotInheritable Class EntitySpanishDesigner
        Inherits EntityDesignerAttribute

        Public Sub New(ByVal Title As String)
            MyBase.New(Enumerations.EnumLanguage.Spanish, Title)
        End Sub

    End Class

End Namespace

File: Common/InMotionGIT.Common/MetadataCatalogAttributes/EntitySpanishDesigner.cs

namespace InMotionGIT.Common.Attributes
{

    /// <summary>
    /// Define el titulo o nombre completo en español  asociado a la clase.
    /// </summary>
    /// <remarks></remarks>
    public sealed class EntitySpanishDesigner : EntityDesignerAttribute
    {

        public EntitySpanishDesigner(string Title) : base((int)Enumerations.EnumLanguage.Spanish, Title)
        {
        }

    }

}